lib/hanami/cli/commands/app/generate/action.rb

```diff
@@ -51,6 +54,8 @@ class Action < App::Command
               default: DEFAULT_SKIP_ROUTE,
               desc: "Skip route generation"
             option :slice, required: false, desc: "Slice name"
+            option :template, required: false, type: :string, default: DEFAULT_TEMPLATE,
```

Comment:
I used `template` because this was mentioned in the original issue and apparently it was named like this in Hanami 1 times. But maybe `template-engine` would be more descriptive? Although a bit long.